examples/compose/e2e.yaml

```diff
@@ -51,6 +51,12 @@ verify:
     count: 10
     # the interval between two attempts, e.g. 10s, 1m.
     interval: 10s
+
+  setting:
+    # whether by fail-fast or not, e.g. true, false.
+    fail-fast: true
```

Comment:
Let's put `fail-fast` under the `verify` instead of `setting`. The subsequent `concurrency` should be the same.

Besides, the comment can be polished, how about:
```yaml
# when a case fails, whether to stop verifying other cases. This property defaults to false.
fail-fast: false
```

Reply:
ok! I will change it later!